docs/sources/clients/promtail/stages/drop.md

```diff
@@ -0,0 +1,179 @@
+---
+title: drop
+---
+# `drop` stage
+
+The `drop` stage is a filtering stage that lets you drop logs based on several options. 
+
+It's important to note that if you provide multiple options they will be treated like an AND clause,
+where each option has to be true to drop the log.
+
+If you wish to drop with an OR clause, specify multiple drop stages.
+
+There are examples below to help explain. 
+
+## Schema
+
+```yaml
+drop:
+  # Name from extracted data to parse. If empty, uses the log message.
+  [source: <string>]
+  
+  # RE2 regular expression, if source is provided the regex will attempt to match the source
+  # If no source is provided, the regex will attempt to match the log line
+  # If the provided regex matches the log line or a provided source, the line will be dropped. 
+  [expression: <string>]
+
+  # value can only be specified when source is specified. It is an error to specify value and regex.
+  # If the value provided is an exact match for the `source` the line will be dropped.
+  [value: <string>]
+
+  # older_than will be parsed as a Go duration: https://golang.org/pkg/time/#ParseDuration
+  # If the log line timestamp is older than the current time minus the provided duration it will be dropped.
+  [older_than: <duration>]
+
+  # longer_than is a value in bytes, any log line longer than this value will be dropped.
+  [longer_than: <int>]
+```
+
+## Examples
+
+### Simple Drops
+
+Simple `drop` stage configurations only specify one of the options, or 2 options when using the `source` option.
```

Comment:
```suggestion
Simple `drop` stage configurations only specify one of the options, or two options when using the `source` option.
```

Reply:
Per the [Documentation style guide](https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-style-guide.md), write out numbers one through ten.